.eslintrc

```diff
@@ -248,6 +251,22 @@
     ],
     "wrap-regex": 0,
     "no-var": 0,
-    "max-len": [2, 80, 4]
+    "max-len": [2, 80, 4],
+    "flowtype/define-flow-type": 2,
+    "flowtype/space-after-type-colon": [
+      1,
```

Comment:
This was relaxed to a warning due to this issue: https://github.com/gajus/eslint-plugin-flowtype/issues/31
(there is one case where a newline follows the colon to get around `max-length`)